opentelemetry/instrumentation/app/work.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+)
+
+const name = "work"
+
+var (
+	meter         = otel.Meter(name)
+	tracer        = otel.Tracer(name)
+	workHistogram metric.Int64Histogram
+)
+
+func init() {
+	var err error
+	workHistogram, err = meter.Int64Histogram("example.histogram",
+		metric.WithDescription("Sample histogram"),
+		metric.WithUnit("1"))
+
+	if err != nil {
+		panic(err)
+	}
+}
+
+// doWork simulates a some job being triggerred in response to an API call to the server.
+// This function computes 10 random values and records them into a histogram which can be
+// later visualized as a distribution.
+func doWork(ctx context.Context, host string) time.Duration {
```

Comment:
Lets come up with a better name (e.g. randomSleep() or computeSubrequests()) that says what it is actually doing